test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/SemaphoreTest.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.concurrent
+
+import java.util.{Collection, Collections}
+import java.util.concurrent.Semaphore
+
+import org.junit.Assert._
+import org.junit.Test
+
+import org.scalajs.testsuite.utils.AssertThrows._
+
+class SemaphoreTest {
+
+  @Test def ctorUnfair(): Unit = {
+    val sem = new Semaphore(1)
+    assertFalse(sem.isFair)
```

Comment:
We usually use `()` when calling JDK methods, don't we?
```suggestion
    assertFalse(sem.isFair())
```
Same for `availablePermits()`